script.js

```diff
@@ -172,3 +172,22 @@ seeProjectBtn.forEach((btn, index) => {
     });
   });
 });
+
+const form = document.querySelector('form');
+const email = document.querySelector('#email');
+const emailError = document.querySelector('.error');
+
+if (form) {
+  form.addEventListener('submit', (event) => {
+    const validEmail = email.value.toLowerCase();
+    emailError.textContent = '';
+    if (email.value !== validEmail) {
+      event.preventDefault();
+      const message = document.createTextNode('Email Must be lowercase');
+      emailError.appendChild(message);
+      emailError.style.color = 'red';
+      return false;
+    }
+    return true;
+  });
+}
```

Comment:
[OPTIONAL]
- [ ] The `if(form)` block doesn't have any logical meaning of being there. means it can be deleted and your codebase will still function as usual. You may consider implementing the KISS (Keep it Smart and Simple)principle of clean code.
- [ ] Also for a better user experience, you may consider adjusting your error color since it's not very going very good with the background color of your contact section.

Reply:
Thank you, I will work on it